nano/node/active_transactions.cpp

```diff
@@ -587,7 +529,19 @@ bool nano::active_transactions::add (std::shared_ptr<nano::block> block_a, bool
 			uint64_t difficulty (0);
 			error = nano::work_validate (*block_a, &difficulty);
 			release_assert (!error);
-			roots.get<tag_root> ().emplace (nano::conflict_info{ root, difficulty, difficulty, election });
+			auto inserted = roots.get<tag_difficulty> ().emplace (nano::conflict_info{ root, difficulty, difficulty, election });
+			if (inserted.first != roots.get<tag_difficulty> ().begin ())
+			{
+				--inserted.first;
+				if (!inserted.first->election->idle ())
```

Comment:
This is not immediately obvious at first glance, but as far as I understand it will start the election as passive as long as the election just before it is not idle? Could you add a comment on what's happening, is there any case where an election will be idle?